File: College-Sports-WebApp/ClientApp/src/api/models/league.ts
/* tslint:disable */
/* eslint-disable */
import { LeagueSeason } from '../models/league-season';
import { Logo } from '../models/logo';
export interface League {
  abbreviation?: string | null;
  calendar?: Array<string> | null;
  calendarEndDate?: string | null;
  calendarIsWhitelist?: boolean;
  calendarStartDate?: string | null;
  calendarType?: string | null;
  href?: string | null;
  id?: number;
  logos?: Array<Logo> | null;
  midsizeName?: string | null;
  name?: string | null;
  season?: LeagueSeason;
  slug?: string | null;
  uid?: string | null;
}

File: College-Sports-Domain/DTO/League.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;
namespace College_Sports_WebApp.Database.Models{ 

    public class League
    {
        [JsonNumberHandling(JsonNumberHandling.AllowReadingFromString)]
        public int id { get; set; }
        public string uid { get; set; }
        public string name { get; set; }
        public string abbreviation { get; set; }
        public string midsizeName { get; set; }
        public string slug { get; set; }
        public LeagueSeason season { get; set; }
        public List<Logo> logos { get; set; }
        public string calendarType { get; set; }
        public bool calendarIsWhitelist { get; set; }
        public string calendarStartDate { get; set; }
        public string calendarEndDate { get; set; }
        public List<string> calendar { get; set; }
        public string href { get; set; }
    }

}